app/stac_api/management/commands/calculate_extent.py

```diff
@@ -0,0 +1,108 @@
+import time
+
+from django.core.management.base import BaseCommand
+from django.core.management.base import CommandParser
+from django.db import connection
+
+from stac_api.utils import CommandHandler
+
+
+def boolean_input(question, default=None):
+    result = input(f"{question}")
+    if not result and default is not None:
+        return default
+    return len(result) > 0 and result[0].lower() == "y"
+
+
+class Handler(CommandHandler):
+
+    def run(self):
+        self.print_success('Runing command to update collection extents')
+        with connection.cursor() as cursor:
+            # print(self.options)
+            if self.options['all']:
+                cursor.execute("SELECT id FROM stac_api_collection")
+            else:
+                cursor.execute("SELECT id FROM stac_api_collection WHERE extent_out_of_sync = TRUE")
+            collections = cursor.fetchall()
```

Comment:
Why using the cursor and not django orm?

Reply:
I started with the update command and then also put the select into the cursor without much thought.
Changed it to have the select commands done by orm as I feel it looks nicer. But kept the actual update command in cursor as I don't know how that could be done using the orm.